planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp

```diff
@@ -118,9 +121,9 @@ double getLateralShift(const LaneChangePath & path);
 
 CandidateOutput assignToCandidate(
   const LaneChangePath & lane_change_path, const Point & ego_position);
-std::optional<lanelet::ConstLanelet> getLaneChangeTargetLane(
-  const RouteHandler & route_handler, const lanelet::ConstLanelets & current_lanes,
-  const LaneChangeModuleType type, const Direction & direction);
+
+std::optional<lanelet::ConstLanelet> get_lane_change_target_lane(
+  const CommonDataPtr & common_data_ptr, const lanelet::ConstLanelets & current_lanes);
```

Comment:
Is there any reason for using a pointer here instead of directly passing `CommonData` by reference ?

Reply:
@maxime-clem 
I would like to standardize the input to avoid mixing argument types within the lane change functions. For example, some functions use `CommonData`, while others use `CommonDataPtr`.

Are there any concerns with this implementation?

Reply:
Note: discussion with @maxime-clem 
In this function, there's no reason to use pointer as input argument, since we didn't even check for any null values.

So for standardization, we will make another PR to refactor the usage of CommonDataPtr